### ISSUE TITLE
看了代码感觉loss是training loss 是吗？

### ISSUE THREAD

**Issue by myrainbowandsky** [nl8590687/ASRT_SpeechRecognition#107]
如何显示validation loss呢？

**Comment by 228311995** [comment]
对的，要显示validation loss可以自己控制训练步数，每训练若干步调用keras.backend.ctc_batch_cost（）计算loss。

**Comment by myrainbowandsky** [comment]
> 对的，要显示validation loss可以自己控制训练步数，每训练若干步调用keras.backend.ctc_batch_cost（）计算loss。

我试试

**Comment by nl8590687** [comment]
在重构的新版代码中，已经预留了callback参数，可以自行传入callback函数来计算验证集loss了
